clang/test/SemaSYCL/sycl-dllimport-dllexport.cpp

```diff
@@ -0,0 +1,48 @@
+// RUN: %clang_cc1 -triple spir64-unknown-linux-sycldevice -fsycl-is-device -fms-extensions -DTEST %s -o /dev/null 2>&1 | FileCheck %s --check-prefixes CHECK
```

Comment:
I think `-aux-triple` setting host triple is missing.
Isn't this important to set?
Depending on what is the host triple compile should behave differently compiling the host part of the application. Right?

Reply:
That is good one.  I don't know what that.  I was trying to get target, don't know this -aux-triple exist and was told using SyclDevice